src/pyproject_fmt/formatter/tools.py

```diff
@@ -27,18 +27,31 @@ def fmt_tools(parsed: TOMLDocument, conf: Config) -> None:  # noqa: ARG001
         table = tools[tool]
         ensure_newline_at_end(cast(Table, table))
     order = [
+        # Build backends
         "poetry",
         "setuptools",
+        "distutils",
         "setuptools_scm",
         "hatch",
+        "flit",
```

Comment:
Add docformatter too please, and please update the test we have to validate this.

Reply:
Ahh, I looked very briefly for a test, should have looked harder. Sure, will do tonight or tomorrow.